relay/network_graph/graph.py

```diff
@@ -334,28 +312,35 @@ def dump(self):
                              'Creditline BA': account.reverse_creditline})
         return output.getvalue()
 
-    def _cost_func_fast_reverse(self, b, a, data, value):
+    def _get_fee(self, data, u, v, value):
         """computes the cost (i.e. the fee) for transferring value from a to b
 
         returns None if the transfer would exceed the creditline.
         """
         # this func should be as fast as possible, as it's called often
         # don't use Account which allocs memory
         # this function calculate the interests to take into account an updated balance
-        pre_balance = balance_with_interest_estimation(data)
-        if a < b:
-            creditline = data[creditline_ba]
-        else:
-            pre_balance = -pre_balance
-            creditline = data[creditline_ab]
+        pre_balance = self._get_balance(data, u, v)
         cost = imbalance_fee(self.capacity_imbalance_fee_divisor, pre_balance, value)
+
+        creditline = get_creditline(data, v, u)
         assert cost >= 0
         post_balance = pre_balance - value - cost
         assert post_balance <= pre_balance
         if -post_balance > creditline:
             return None  # no valid path
         return cost
 
+    def _get_capacity(self, data, u, v):  # gets the capacity from u to v
+        balance_with_interests = self._get_balance(data, u, v)
+        return get_creditline(data, v, u) + balance_with_interests
+
+    def _get_balance(self, data, u, v):
```

Comment:
Should we rename that function to represent that it gets the balance + interest on current server time?
Something like "_get_balance_with_interests_on_current_time()" or "_get_balance_with_interests()"

Reply:
yeah probably makes sense